packages/env/README.md

```diff
@@ -253,23 +253,25 @@ The start command installs and initializes the WordPress environment, which incl
 ```sh
 wp-env start
 
-Starts WordPress for development on port 8888 (override with WP_ENV_PORT) and
-tests on port 8889 (override with WP_ENV_TESTS_PORT). The current working
-directory must be a WordPress installation, a plugin, a theme, or contain a
-.wp-env.json file. After first install, use the '--update' flag to download
-updates to mapped sources and to re-apply WordPress configuration options.
+Starts WordPress for development on port 8888 (​http://localhost:8888​)
+(override with WP_ENV_PORT) and tests on port 8889 (​http://localhost:8889​)
+(override with WP_ENV_TESTS_PORT). The current working directory must be a
+WordPress installation, a plugin, a theme, or contain a .wp-env.json file. After
+first install, use the '--update' flag to download updates to mapped sources and
+to re-apply WordPress configuration options.
 
 Options:
-  --help     Show help                                                 [boolean]
-  --version  Show version number                                       [boolean]
-  --debug    Enable debug output.                     [boolean] [default: false]
-  --update   Download source updates and apply WordPress configuration.
+  --debug            Enable debug output.             [boolean] [default: false]
+  --update           Download source updates and apply WordPress configuration.
                                                       [boolean] [default: false]
-  --xdebug   Enables Xdebug. If not passed, Xdebug is turned off. If no modes
-             are set, uses "debug". You may set multiple Xdebug modes by passing
-             them in a comma-separated list: `--xdebug=develop,coverage`. See
-             https://xdebug.org/docs/all_settings#mode for information about
-             Xdebug modes.                                              [string]
+  --xdebug           Enables Xdebug. If not passed, Xdebug is turned off. If no
+                     modes are set, uses "debug". You may set multiple Xdebug
+                     modes by passing them in a comma-separated list:
+                     `--xdebug=develop,coverage`. See
+                     https://xdebug.org/docs/all_settings#mode for information
+                     about Xdebug modes.                                [string]
+  --execute-scripts  Execute any configured lifecycle scripts.
```

Comment:
Would `--scripts` be descriptive enough? It's clear to me, and easier to type.

Reply:
@ObliviousHarmony thanks! That was fast.

Reply:
I _really_ want to get this merged, haha.